packages/flutter/lib/src/material/refresh_indicator.dart

```diff
@@ -47,6 +47,17 @@ enum _RefreshIndicatorMode {
   canceled, // Animating the indicator's fade-out after not arming.
 }
 
+/// Used to configure the [RefreshIndicator] trigger mode.
+enum RefreshIndicatorTriggerMode {
+  /// The indicator can trigger when the child's [Scrollable] descendant overscrolls
```

Comment:
suggestion: The indicator can be triggered regardless of the scroll position of the [Scrollable] when the drag starts.